addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -31,530 +20,95 @@ class ComposerTextInput extends Component {
          * Max frequency for searching for mention suggestions. Useful to
          * limit amount of RPC while user types something that starts a mention.
          */
-        this.MENTION_THROTTLE = 200;
         this.storeDispatch = useDispatch();
         this.storeGetters = useGetters();
-        this.storeProps = useStore(state => {
+        this.storeProps = useStore((state, props) => {
+            const composer = state.composers[props.composerLocalId];
             return {
                 isMobile: state.isMobile,
+                composer,
```

Comment:
not a big deal, could be done in one line here: `composer: state.composers[props.composerLocalId],`

Reply:
Done